<launch>
  <node pkg="inertiallabs_ins" type="il_ins" name="INS" output="screen" respawn="true">
   <param name="ins_url" type="string" value="serial:/dev/ttyUSB0:9600" />
   <param name="ins_output_format" type="int" value="88" />
  </node>
</launch>
